server/core/src/main/java/dev/slimevr/tracking/trackers/Tracker.kt

```diff
@@ -436,4 +437,13 @@ class Tracker @JvmOverloads constructor(
 	fun resetFilteringQuats(reference: Quaternion) {
 		filteringHandler.resetMovingAverage(getAdjustedRotation(), reference)
 	}
+
+	companion object {
+		/**
+		 * Changes from IMU axis to OpenGL/SteamVR axis
+		 */
+		fun axisOffset(v: Vector3): Vector3 = Vector3(v.x, v.z, -v.y)
+		private val rotationOffset = Quaternion(INV_SQRT_TWO, -INV_SQRT_TWO, 0f, 0f)
```

Comment:
The axes may need to be rotated along the device before remap
```suggestion
		fun axisRemap(v: Vector3): Vector3 = Vector3(v.x, v.z, -v.y)
		private val rotationOffset = Quaternion(INV_SQRT_TWO, -INV_SQRT_TWO, 0f, 0f)
		fun axisOffset(v: Vector3): Vector3 = axisRemap(rotationOffset.sandwich(v))
```

Reply:
`axisOffset` is an optimized version for `rotationOffset.sandwich(v)`
It either one or another, they are the same.

BTW, I would recommend to 
- store only normalized rotations
- use a single rotation function, which needs ~15 multiplications, or at least use two quaternion multiplications. Current `sandwich` code uses quaternion inversion, which us slow. With current state of Koltin you will not feel the difference. However, in the future, with Project Valhalla, Quaternion can be optimized a lot, so the division will be bottleneck. I am thinking about SlimeVR server running on lightweight XR headsets directly, not PC.